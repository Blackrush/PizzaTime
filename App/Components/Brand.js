import React from 'react'
import {
  StyleSheet,
  View,
  Image,
  Text,
} from 'react-native'


const Brand = (props) => {
  return (
    <View style={[styles.header, props.style]}>
      <Image source={require('../Assets/trakt.png')}
             resizeMode="stretch"
             style={styles.image}
      />
      <View style={styles.headerTextContainer}>
        <Text style={[styles.headerText, {color: '#D33E1A'}]}>Pizza</Text>
        <Text style={[styles.headerText, {color: '#F6BA57'}]}>Time</Text>
      </View>
    </View>
  )
}


const styles = StyleSheet.create({
  header: {
    alignItems: 'center',
  },
  image: {
    height: 90,
    width: 90,
  },
  headerTextContainer: {
    flexDirection: 'row'
  },
  headerText: {
    textAlign: 'center',
    fontSize: 36,
    fontFamily: 'GillSans-Light',
  },
})


export default Brand