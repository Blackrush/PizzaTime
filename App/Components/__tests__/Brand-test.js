import React from 'react'
import Brand from '../Brand'
import { create as render } from 'react-test-renderer'

test('Brand renders correctly', () => {
  const tree = render(
    <Brand/>
  ).toJSON()

  expect(tree).toMatchSnapshot()
})
