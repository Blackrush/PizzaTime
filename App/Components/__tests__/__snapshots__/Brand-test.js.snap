exports[`test Brand renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center"
      },
      undefined
    ]
  }>
  <Image
    resizeMode="stretch"
    source={1}
    style={
      Object {
        "height": 90,
        "width": 90
      }
    } />
  <View
    style={
      Object {
        "flexDirection": "row"
      }
    }>
    <Text
      style={
        Array [
          Object {
            "fontFamily": "GillSans-Light",
            "fontSize": 36,
            "textAlign": "center"
          },
          Object {
            "color": "#D33E1A"
          }
        ]
      }>
      Pizza
    </Text>
    <Text
      style={
        Array [
          Object {
            "fontFamily": "GillSans-Light",
            "fontSize": 36,
            "textAlign": "center"
          },
          Object {
            "color": "#F6BA57"
          }
        ]
      }>
      Time
    </Text>
  </View>
</View>
`;
